### ISSUE TITLE
Library and Preferences closed when pressing ESC in insert mode 

### ISSUE THREAD

**Issue by vdnhi** [vimdotmd/vimdotmd#24]
I am typing and press ESC with expecting behavior to switch vim mode, but 2 settings Library and Preferences also close (unexpected behavior).
Is it an intentional behavior?
![Screencast from 02-10-2020 16_11_08](https://user-images.githubusercontent.com/9262617/94907280-1bb9d300-04ca-11eb-93b6-2c74c74f381c.gif)

**Comment by thien-do** [comment]
Yes at the moment it's expected behavior... It let the user to quickly going back to the editor at anytime.

It'd be great if you can tell us why you need to keep the Library and Preferences open while writing?

**Comment by vdnhi** [comment]
I'm not sure that I always need Preferences when editing or not, but with Library, when I opened a repository and editing some files on it. I want to see what is the file I am editing and with Library always open help me to move between 2 different files easier than have to click again on the Library button and choose another file to edit.

**Comment by thien-do** [comment]
That makes sense! We will avoid closing the Library when pressing ESC in the editor

**Comment by thien-do** [comment]
Btw, in the upcoming release we will also show the current file title on the tab's title. It would make it's easier for you to see what is the current file

**Comment by vdnhi** [comment]
The Library now is showing all the files in the repository. I think it is better to show only markdown files (like typing `*.md` in the Search box, but we can't make sure that everyone knows about how to filter markdown files like that) because `vim.md` is a markdown editor. Another option is showing whole the repository but with folder structure. How do you feel about it?

**Comment by thien-do** [comment]
@nomisrevol thanks for the suggestion em :+1: In fact we are looking into both:
- https://github.com/vimdotmd/vimdotmd/issues/26 To show only Markdown files
- https://github.com/vimdotmd/vimdotmd/issues/6 To use a tree instead of a flat list for the Library
